src/Dotnet.Script.Tests/ScriptExecutionTests.cs

```diff
@@ -75,6 +82,15 @@ public static void ShouldNotPassUnEscapedKnownArgumentToScript()
             Assert.DoesNotContain("-v", result.output);            
         }
 
+        [Fact]
+        public static void ShouldPropagateReturnValue()
+        {
+            var result = Execute($"{Path.Combine("ReturnValue", "ReturnValue.csx")}");
```

Comment:
since the errors are 1, let's use a different return value for this "happy path" test 😊

Reply:
I agree, lets use 42 :)